torch/csrc/autograd/VariableTypeUtils.h

```diff
@@ -121,6 +121,9 @@ struct Flatten : IterArgs<Flatten> {
   Flatten(variable_list& out) : out(out) {}
   variable_list& out;
   void operator()(const at::Tensor& x) { out.emplace_back(x); }
+  void operator()(const c10::optional<at::Tensor>& x) {
+    if (x.has_value()) out.emplace_back(x.value());
+  }
```

Comment:
Nit: Even trivial if statement body should be wrapped by curly brackets:
```suggestion
  void operator()(const c10::optional<at::Tensor>& x) {
    if (x.has_value()) {
      out.emplace_back(x.value());
    }  
  }
```

Reply:
Thank you for the suggestion. Will update it,

Reply:
removed changes to VariableTypeUtils.h in this PR.